etc/docker-scripts/bootstrapper.py

```diff
@@ -141,20 +142,35 @@ def put_file_object_store(client, bucket_name, file_to_upload, subdir):
           .format(str(input_params['input']),
                   str(input_params['output'])))
 
-    papermill.execute_notebook(
-        input_params['input'],
-        input_params['output'],
-        kernel_name="python3"
-        # parameters=
-    )
-    output_html_file = notebook_to_html(input_params['output'], input_params['output-html'])
-    print("Uploading Results back to Object Storage")
-    put_file_object_store(cos_client, input_params["bucket"], output_html_file, cos_dir_pre)
-    put_file_object_store(cos_client, input_params["bucket"], input_params['output'], cos_dir_pre)
-
-    output_list = input_params['pipeline-outputs'].split(",")
-    for file in output_list:
-        if file != 'None':
-            put_file_object_store(cos_client, input_params['bucket'], file, cos_dir_pre)
+    try:
+        papermill.execute_notebook(
+            input_params['input'],
+            input_params['output'],
+            kernel_name="python3"
+            # parameters=
+        )
+        output_html_file = notebook_to_html(input_params['output'], input_params['output-html'])
+        print("Uploading Results back to Object Storage")
+        put_file_object_store(cos_client, input_params["bucket"], output_html_file, cos_dir_pre)
+        put_file_object_store(cos_client, input_params["bucket"], input_params['output'], cos_dir_pre)
+
+        output_list = input_params['pipeline-outputs'].split(",")
+        for file in output_list:
+            if file != 'None':
+                put_file_object_store(cos_client, input_params['bucket'], file, cos_dir_pre)
+    except:
+        output_error_html = input_params['output-html'].replace('.html', '-error.html')
+        output_html_file = notebook_to_html(input_params['output'], output_error_html)
```

Comment:
Could you please add a comment at the top of the exception handler that describes what is going on.  It took me a minute or two that the difference between the try and except blocks is that the latter is going and producing information about the failure in a similar manner as the non error case.  I think it would be helpful to readers/maintainers to understand why the exception handling is doing what it's doing.